assets/airports/krdu.json

```diff
@@ -0,0 +1,4178 @@
+            "rwy": {
+                "KRDU5L": ["#052"],
+                "KRDU5R": ["#052"],
+                "KRDU14": ["#142"],
+                "KRDU23L": ["#232"],
+                "KRDU23R": ["#232"],
+                "KRDU32": ["#322"]
+            },
+            "body": ["RDU", "EVIGY", "FAY"],
+            "exitPoints": {
+                "CAE": ["CAE"],
+                "CRE": ["CRE"],
+                "FLO": ["FLO"]
+            },
+            "draw": [
+                ["RDU", "EVIGY", "FAY"],
+                ["CAE*"],
+                ["CRE*"],
+                ["FLO*"]
+            ]
+        },
+        "HOOKZ2": {
+            "icao": "HOOKZ2",
+            "name": "Hooks Two",
+            "rwy": {
+                "KRDU5L": ["#052"],
+                "KRDU5R": ["#052"],
+                "KRDU14": ["#142"],
+                "KRDU23L": ["#232"],
+                "KRDU23R": ["#232"],
+                "KRDU32": ["#322"]
+            },
+            "body": ["FITON", "HOOKZ"],
+            "exitPoints": {
+                "FAY": ["FAY"]
+            },
+            "draw": [
+                ["FITON", "HOOKZ", "FAY*"]
+            ]
+        },
+        "HURIC2": {
+            "icao": "HURIC2",
+            "name": "Huric Two",
+            "rwy": {
+                "KRDU5L": ["#052"],
+                "KRDU5R": ["#052"],
+                "KRDU14": ["#142"],
+                "KRDU23L": ["#232"],
+                "KRDU23R": ["#232"],
+                "KRDU32": ["#322"]
+            },
+            "body": ["STRMY", "HURIC"],
+            "exitPoints": {
+                "TYI": ["TYI"]
+            },
+            "draw": [
+                ["STRMY", "HURIC", "TYI*"]
+            ]
+        },
+        "LWOOD2": {
+            "icao": "LWOOD2",
+            "name": "Lwood Two",
+            "rwy": {
+                "KRDU5L": ["#052"],
+                "KRDU5R": ["#052"],
+                "KRDU23L": ["#232"],
+                "KRDU23R": ["#232"]
+            },
+            "body": ["FOCAL", "EPOCH", "LWOOD"],
+            "exitPoints": {
+                "PSK": ["OPUSS", "PSK"],
+                "ROA": ["ROA"]
+            },
+            "draw": [
+                ["FOCAL", "EPOCH", "LWOOD", "ROA*"],
+                ["LWOOD", "OPUSS", "PSK*"]
+            ]
+        },
+        "OXFRD2": {
+            "icao": "OXFRD2",
+            "name": "Oxford Two",
+            "rwy": {
+                "KRDU5L": ["#052"],
+                "KRDU5R": ["#052"],
+                "KRDU14": ["#142"],
+                "KRDU23L": ["#232"],
+                "KRDU23R": ["#232"],
+                "KRDU32": ["#322"]
+            },
+            "body": ["PEAVY", "OXFRD", "HARVY", "CARML"],
+            "exitPoints": {
+                "FAK": ["FAK"]
+            },
+            "draw": [
+                ["PEAVY", "OXFRD", "HARVY", "CARML", "FAK*"]
+            ]
+        },
+        "PACK7": {
+            "icao": "PACK7",
```

Comment:
`PACKK7`

Reply:
The ICAO code for the procedure is actually `PACK7` as weird as that is:
![pack](https://user-images.githubusercontent.com/13125311/36121416-ffcb8c34-103d-11e8-8609-a1d5a6d2526b.PNG)